package.nls.json

```diff
@@ -147,6 +156,8 @@
     "cmake-tools.configuration.cmake.configureEnvironment.description": "Environment variables to pass to CMake during configure. When using CMake Presets, these are temporarily added to the environment provided by the active configure preset.",
     "cmake-tools.configuration.cmake.buildEnvironment.description": "Environment variables to pass to CMake during build. When using CMake Presets, these are temporarily added to the environment provided by the active build preset.",
     "cmake-tools.configuration.cmake.testEnvironment.description": "Environment variables to pass to CTest. When using CMake Presets, these are temporarily added to the environment provided by the active test preset.",
+    "cmake-tools.configuration.cmake.cpackArgs.description": "Additional arguments to pass to CPack. When using CMake Presets, these arguments are temporarily added to the arguments provided by the active package preset.",
+    "cmake-tools.configuration.cmake.packEnvironment.description": "Environment variables to set when packaging with CPack. When using CMake Presets, these are temporarily added to the environment used for CPack runs.",
```

Comment:
should this be `cpackEnvironment`? In order to get closer to `cpackArgs`?

Reply:
For testing, I saw ctestArgs and testEnvironment. That's why I went with cpackArgs and packEnvironment. But I agree cpackEnvironment fits better.